hw-35.ts

```diff
@@ -0,0 +1,101 @@
+type User = {
+    name: string,
+    phone: string,
+    email: string,
+    animals?: string[],
+    cars?: string[],
+    hasChildren: boolean,
+    hasEducation: boolean
+}
+
+const users: User[] = [
+        {
+            name: "Harry Felton",
+            phone: "(09) 897 33 33",
+            email: "felton@gmail.com",
+            animals: ["cat"],
+            cars: ["bmw"],
+            hasChildren: false,
+            hasEducation: true
+            
+        },
+        {
+            name: "May Sender",
+            phone: "(09) 117 33 33",
+            email: "sender22@gmail.com",
+            hasChildren: true,
+            hasEducation: true
+        },
+        {
+            name: "Henry Ford",
+            phone: "(09) 999 93 23",
+            email: "ford0@gmail.com",
+            cars: ["bmw", "audi"],
+            hasChildren: true,
+            hasEducation: false
+        }
+    ]
+
+// Task 1:
+const getUsernameString = (arr) => {
+    const nameArr: string[] = []
+    arr.forEach(element => {
+        nameArr.push(element.name)
```

Comment:
push is a mutation method. avoid using it. 
map will be best for that task
but it's from next HW, so here it's ok